pkg/plugins/generic/generic_plugin.go

```diff
@@ -87,12 +87,31 @@ func (p *GenericPlugin) Apply() error {
 			return nil
 		}
 	}
+
+	// Create a map with all the PFs we will need to configure
+	// we need to create it here before we access the host file system using the chroot function
```

Comment:
It's clear to my why do we need this logic but it doesn't look good architecture to have this map and still use SkipConfigVf function in other places.

Reply:
done we use the same function on both places